netci.groovy

```diff
@@ -109,7 +109,7 @@ wcfUtilities = new WcfUtilities()
     
```

Comment:
Line 101: 

If we want to be consistent with the netci.groovy in corefx, consider changing all 'os' to 'osName'. I don't think it's necessary but might help with diffing (though we are pretty diverged at this point)